AsyncDisplayKit/ASTextNode.h

```diff
@@ -74,6 +74,13 @@ typedef NS_ENUM(NSUInteger, ASTextNodeHighlightStyle) {
 @property (nonatomic, assign) NSUInteger maximumNumberOfLines;
 
 /**
+ * @abstract flexShrink is YES by default for ASTextNode, allowing it to wrap to
+ * multple lines when contained in stacks.
+ * @default YES
+ */
+@property (nonatomic, assign) BOOL flexShrink;
```

Comment:
@appleguy @nguyenhuy Why do we need to add this property to ASTextNode? It's already in ASLayoutable.h